### DISCUSSION TITLE
SSL Certificate not renewed on jamulus.io

### DISCUSSION THREAD

**Discussion by anprdev** [jamulussoftware/jamulus#3432] [Bug Investigation]
Hi
https://jamulus.io has a certificate that is not good anymore
Have disable VPN and this is also not the issue.
Reboot the Computer also not helping.
Try a Raspberry PI also the same.

On PI is http://jamulus.io working no https

I hoop this is not a problem.

**Comment by ann0see** [comment]
@anprdev thanks for notifying us. We're aware of the issue. 

@gilgongo should know this. I assume that it's a problem with cloudflare or - more likely - GitHub. The settings show a DNS error. Maybe something needs to be setup in the jamulus.io zone to allow updates of the cert.

**Comment by anprdev** [comment]
SSL Solved I see.
Good

**Reply by ann0see** [reply]
Yes. We're still checking the root cause and hope that we find a solution that certificate renewal works automatically.

**Reply by ann0see** [reply]
I believe by disabling the cloudflare proxy, let's encrypt renewals should work.

**Reply by gilgongo** [reply]
Yes. And running without the proxy is unlikely to be an issue. We moved to Cloudflare so that we could have multiple people able to manage the domain, which wasn't really the case before.
